tests/components/deconz/test_sensor.py

```diff
@@ -277,10 +211,8 @@ async def test_add_new_sensor(hass):
     gateway.api.event_handler(state_added_event)
     await hass.async_block_till_done()
 
-    assert "sensor.light_level_sensor" in gateway.deconz_ids
-
-    light_level_sensor = hass.states.get("sensor.light_level_sensor")
-    assert light_level_sensor.state == "999.8"
+    assert len(hass.states.async_all()) == 1
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6271139)